src/pages/publication/submission_tab/submission_journal.tsx

```diff
@@ -2,27 +2,29 @@ import LabelContentRow from '../../../components/LabelContentRow';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { FormikProps, useFormikContext } from 'formik';
-import { Publication } from '../../../types/publication.types';
+import { Publication, PagesRange } from '../../../types/publication.types';
 
 const SubmissionJournalPublication: React.FC = () => {
   const { t } = useTranslation('publication');
   const { values }: FormikProps<Publication> = useFormikContext();
 
   const {
     reference: {
-      publicationInstance: { type, volume, issue, pages, articleNumber, peerReviewed },
+      publicationInstance: { type, volume, issue, articleNumber, peerReviewed },
```

Comment:
virker ikke med
```suggestion
      publicationInstance: { type, volume, issue, pages as PagesRange, articleNumber, peerReviewed },
```
?

Reply:
nope 😕